utils/src/mkfs.c

```diff
@@ -395,22 +395,22 @@ static bool valid_quorum_slots(struct scoutfs_quorum_slot *slots)
 	int j;
 
 	for (i = 0; i < SCOUTFS_QUORUM_MAX_SLOTS; i++) {
-		if (slots[i].addr.addr == 0)
+		if (slots[i].addr.v4.addr == 0)
 			continue;
 
```

Comment:
The only persistent address inputs are the config slots.  This should verify that the family is known (only v4 today) and fail if it sees an unrecognized family.  This will ensure that the structures are already consistent when they're found for the first time by code that supports both v4 and v6.